lib/cassanity/config.rb

```diff
@@ -0,0 +1,22 @@
+
+require 'ostruct'
+require 'yaml'
+require 'singleton'
+require 'erb'
+
+module Cassanity
+  class Config < OpenStruct
```

Comment:
I don't really like open struct. I'd rather define each of the config options. I think it makes things more explicit and discoverable.